marklogic-data-hub-spark-connector/src/test/resources/custom-job-endpoints/initializeRead.sjs

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
See my note above for testing - this should return a static schema and partition object. I believe parameterizedPlan can be "{}" because your custom read endpoint won't use it.